accepted/future-releases/constructor-tearoffs/feature-specification.md

```diff
@@ -394,6 +394,12 @@ Currently you can do instantiated tear-offs of *instance* methods. We restrict t
 
 That makes it a compile-time error to *explicitly* instantiate the `call` method of an expression with a function type or of type `Function`, and the tear-off of a `call`  method of a function type is not subject to implicit instantiation (so the tear-off is always generic, even if the context type requires it not to be, which is then guaranteed to introduce a type error).
 
+### Constructors in abstract classes
+
+A compile-time error occurs if a constructor tear-off denotes a generative constructor declared in an abstract class.
+
+*A generative constructor declared in an abstract class may well be executed as part of the execution of a constructor tear-off, via a superinitializer. But it cannot be torn off on its own, because the execution of a constructor tear-off is an instance creation, and abstract classes do not have instances. There is no similar constraint on a factory constructor.*
```

Comment:
I'd prefer to not start the sentence with "But".
Maybe "It cannot, however, be ...."?

Reply:
Done!